ercc/Makefile

```diff
@@ -18,8 +18,8 @@ attestation-clean:
 	$(MAKE) -C $(ATTESTATION_DIR) clean
 
 ercc: $(ATTESTATION_PROVIDER)
+	# ercc's binary is created here and referenced by external launcher for run-on-host mode
```

Comment:
You caught something here. I was getting `ercc` from both the peer cli (for packaging it) and the external launcher `build`. Clearly the latter is inappropriate since I had already put the binary in the package.

tltr: only peer cli references ercc binary, and puts it in package